openc3-cmd-tlm-api/Dockerfile

```diff
@@ -8,13 +8,8 @@ COPY Gemfile ./
 
 USER root
 
-RUN apk add --virtual .build-dependencies \
-    build-base \
-    ruby-dev \
-    libressl-dev \
```

Comment:
How did all this go away?

Reply:
Its already in the openc3-ruby base container.